### DISCUSSION TITLE
Adding R

### DISCUSSION THREAD

**Discussion by jan-casas** [dotnet/interactive#2742] [Ideas]
Would it be possible to add R?
Great extension thank you!! ❤️❤️

**Comment by claudiaregio** [comment]
Hi @jan-casas, we are actually in progress of testing out our Python and R integration right now! If you're willing to sign an NDA and provide us some feedback, you can sign up here to get the VSIX and install instructions to try it out: https://forms.office.com/r/UQchfQSGa5

**Comment by wpalaceJHA** [comment]
Hello. I'm curious to know how this feature is coming along. I filled out the form to try and get access to the alpha, but I never received any follow up.

**Reply by wpalaceJHA** [reply]
I see that Python and R integration are now available in prerelease. Can anyone provide some guidance as to how to test this out? I am using VSCode 1.79 Insider release, Jupyter extension v2023.5.1001531100 Pre-release, and Polyglot Notebooks v1.0.4280031 pre-release. However, I do not see any options to change to a Python or R kernel. I am very excited to test out this feature and would appreciate any help.

**Reply by sharoncxu** [reply]
Hi @wpalaceJHA, there's two things you'll need to do to get the Python and R kernel going. First you'll have to open VS Code from the Anaconda prompt. Then once you've got your polyglot notebook open, you'll have to run a connect command to connect to your kernel:
`#!connect jupyter --kernel-name pythonkernel --kernel-spec python3`

The full documentation for how to use the release can be found here: https://github.com/dotnet/interactive/blob/main/docs/jupyter-in-polyglot-notebooks.md
